### PR TITLE
add peaq evm alphanet (still WIP)

### PR DESCRIPTION
**Pull request by Rihyx**
### Description

!!!!
The Peaq EVM integration for XCM is still not complete. These changes are required for MRL to get balances of the parachain.  The XCM will be fixed later.

### Checklist

- [x] If this requires a documentation change, I have created a PR in [moonbeam-docs](https://github.com/moonbeam-foundation/moonbeam-docs) repository.
- [x] If this requires it, I have updated the Readme
- [x] If necessary, I have updated the examples
- [x] I have verified if I need to create/update unit tests
- [x] I have verified if I need to create/update acceptance tests
- [x] If necessary, I have run acceptance tests on this branch in CI

### CODE REVIEW

File: packages/config/src/chains.ts [hunk 1]
```diff
@@ -1261,6 +1261,44 @@ export const peaqAlphanet = new Parachain({
   ws: 'wss://moonbeam.peaq.network',
 });
 
+export const peaqEvmAlphanet = new EvmParachain({
+  assetsData: [
+    {
+      // FIXME: not sure
+      asset: agng,
+      // id: 0,
```
Comment (jpina) [inline packages/config/src/chains.ts:1269]: does this need to be fixed? is the `id` necessary?

File: packages/config/src/configs/moonbaseAlpha.ts [hunk 2]
```diff
@@ -306,6 +307,32 @@ export const moonbaseAlphaConfig = new ChainConfig({
         balance: BalanceBuilder().substrate().system().account(),
       },
     }),
+    new AssetConfig({
+      asset: dev,
+      balance: BalanceBuilder().substrate().system().account(),
+      contract: ContractBuilder().Xtokens().transfer(),
+      destination: peaqEvmAlphanet,
+      destinationFee: {
+        amount: 0.00000001,
```
Comment (jpina) [inline packages/config/src/configs/moonbaseAlpha.ts:316]: do floating point numbers work well or can we get some weird behavior like unwanted rounding or issues during math operations?

File: packages/config/src/configs/peaqEvmAlphanet.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+import { moonbaseAlpha, peaqEvmAlphanet } from '../chains';
+import { AssetConfig } from '../types/AssetConfig';
+import { ChainConfig } from '../types/ChainConfig';
+
+export const peaqEvmAlphanetConfig = new ChainConfig({
+  assets: [
+    new AssetConfig({
+      asset: ftmwh,
+      balance: BalanceBuilder().evm().erc20(),
+      // contract: ContractBuilder().Xtokens().transfer(),
```
Comment (jpina) [inline packages/config/src/configs/peaqEvmAlphanet.ts:12]: left overs?

**Review by ekenigs — DISMISSED** [review]

File: packages/config/src/chains.ts (hunk 1 above)
Comment (Rihyx) [reply packages/config/src/chains.ts:1269]: removed

File: packages/config/src/configs/moonbaseAlpha.ts (hunk 2 above)
Comment (Rihyx) [reply packages/config/src/configs/moonbaseAlpha.ts:316]: This doesn't matter for now and will be fixed later.

File: packages/config/src/configs/peaqEvmAlphanet.ts (hunk 3 above)
Comment (Rihyx) [reply packages/config/src/configs/peaqEvmAlphanet.ts:12]: This doesn't matter for now and will be fixed later.

**Review by ekenigs — APPROVED** [review]

**Pull request merged**